### PR TITLE
adds reporting 101 blog

### PR DESCRIPTION
**Pull request by elfisher**
### Description
Adds blog by @anirudha and myself on 101 material for Reporting.

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+![scheduled reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/create-report-definition.gif){: .img-fluid}
+
+## Development internals
+
+The reporting feature uses a custom [minimal build](https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports/rendering-engine/headless-chrome) of headless-chromium. Chromium is used as the rendering engine to generate visual reports. Below is are multiple flow diagrams for the various actions of Reporting. 
+
+**Creating a report definition**: When a user defines a report in OpenSearch Dashboards, the Reporting Dashboards plugin saves that definition in an index in OpenSearch. 
+
+**Downloading a report:** When a user downloads a report, the OpenSearch Dashboards Reporting plugin fetches the data from OpenSearch. After the data is fetched, the report is generated and downloaded to the users client.
```
Comment (stockholmux) [inline _posts/2021-06-25-feature-highlight-reporting.markdown:50]: What is 'client' in this context? Browser?

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+**Running a reporting job:** When a schedule report begins, data is fetched from OpenSearch, next the OpenSearch Dashboards Reporting plugin generates the report. Once the report is generated, it is sent to the configured destination in the reporting policy and the job status is updated and stored back in OpenSearch.
+
+![flow diagram](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/flow-diagram.png){: .img-fluid}
```
Comment (stockholmux) [inline _posts/2021-06-25-feature-highlight-reporting.markdown:54]: There is a strange border on the right side of this image.

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Community contributions
+
+Reporting was built by multiple contributors from the community (some PRs listed below). We presented the initial Reporting contributions in a community meeting with the [Reports RFC](https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/master/docs/dev/Kibana-Reporting-Design-Proposal.md). Our team went over the initial design and requirements, and our high level plan to develop the feature. After the community meeting, some engineers at Dassault Systems expressed interest in contributing and helped build the CSV functionality. Since the initial release, there have been numerous other community contributions to improve and customize the project (see below). We thank all the contributors who have helped build the Reporting feature!
+
+* [50](https://github.com/opendistro-for-elasticsearch/kibana-reports/pull/50)
```
Comment (stockholmux) [inline _posts/2021-06-25-feature-highlight-reporting.markdown:60]: A list of numbers looks weird - can we also include the type of the PRs?

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+and more...
+
+If you’re interested in contributing please reach out on [GitHub issues](https://github.com/opensearch-project/dashboards-reports/issues) or [the community forum](https://discuss.opendistrocommunity.dev/). The more formal contribution guidelines are documented [here](https://github.com/opensearch-project/dashboards-reports/blob/main/CONTRIBUTING.md). 
+
+## Future project goals
+
+* Create integrations with Alerting, Notebooks and other plugins
+* Enable support for slack, chime, webhooks, pager-duty, and other destinations via the [Notifications plugin](https://github.com/opensearch-project/notifications).
+* Remove the hard dependency on headless-chromium by enabling support for a Java based rendering engine.
```
Comment (stockholmux) [inline _posts/2021-06-25-feature-highlight-reporting.markdown:77]: Java**-**based

**Review by stockholmux — CHANGES_REQUESTED** [review]
I found a few issues with language and one with the image.

Additionally, it seems like the meaning of 'we' drifts around bit throughout the piece. In this context, 'we' should mean the authors only.

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+**How can reports be generated ?**
+Reports are generated manually on OpenSearch Dashboards, via scheduled triggers, or through system triggers like alerts or API calls.
+
+**Anatomy of the Reporting feature on OpenSearch**
+The reporting feature is composed of two plugins; an [OpenSearch Dashboards plugin](https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports) which controls most the [user experience](https://github.com/opensearch-project/dashboards-reports/blob/main/docs/dashboards-reports/ux/OpenSearch-Dashboards-Reporting-UX-documentation.md) and the [OpenSearch plugin](https://github.com/opensearch-project/dashboards-reports/tree/main/reports-scheduler) which provides scheduling features and secures access to user data with OpenSearch Security policies. Our initial request for comments (RFC) for Reporting can be found [here](https://github.com/opensearch-project/dashboards-reports/blob/main/docs/dashboards-reports/dev/OpenSearch-Dashboards-Reporting-Design-Proposal.md).
```
[the remark this answers is not in the document]
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+![Report Anatomy](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/report-anatomy.png){: .img-fluid}
+
+## Quick Start User Guide
+
+### Installation
+
+Reporting comes packaged in with the OpenSearch and OpenSearch Dashboards downloads. You can follow the installation instructions on [opensearch.org/downloads](http://opensearch.org/downloads)
```
[the remark this answers is not in the document]
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

File: _posts/2021-06-25-feature-highlight-reporting.markdown [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+## Quick Start User Guide
+
+### Installation
+
+Reporting comes packaged in with the OpenSearch and OpenSearch Dashboards downloads. You can follow the installation instructions on [opensearch.org/downloads](http://opensearch.org/downloads) 
+
+### How do I create a report?
+
+**Visual reports** are created inside the Visualization or Dashboards experience.
```
[the remark this answers is not in the document]
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

File: _posts/2021-06-25-feature-highlight-reporting.markdown (hunk 1 above)
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

File: _posts/2021-06-25-feature-highlight-reporting.markdown (hunk 3 above)
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

File: _posts/2021-06-25-feature-highlight-reporting.markdown (hunk 4 above)
Comment (elfisher) [reply _posts/2021-06-25-feature-highlight-reporting.markdown]: fixed

**Comment by elfisher** [comment]
@stockholmux, @anirudha and myself addressed the feedback. The dates are also updated to 6/29 Let us know if this looks good now on your end.

**Pull request merged**